public class FunctionUse4406 {

    public void funcUse() {
		org.apache.hadoop.fs.PathIOException pathioexception = new org.apache.hadoop.fs.PathIOException();
		pathioexception.setOperation(null);
    }
}